blogposts/2021/how-to-not-break-a-search-engine-or-what-i-learned-about-unglamorous-engineering.md

```diff
@@ -184,17 +189,17 @@ gone as planned. I'd pulled off my greatest feat of unglamorous engineering.
 Hardly anyone could appreciate how months of effort culminated in an
 incident-free transition. That was, after all, expected. I mean, how do you
 derive appreciation from users, peers, or managers when the point was for no
-one to notice anything significant had changed... when there's no perceptible
+one to notice anything significant had changed; when there's no perceptible
 delta?
 
 ## Value in the unseen and unspoken
 
-There's unglamorous engineering activity in the software all around us. For all
+There's unglamorous engineering in the software all around us. For all
 its lack of recognition, I wish we grasped its value a bit better. I'm reminded
 of a tweet by a former colleague who researched donations for open source
 projects:
 
-<p style="padding: 0px 40px 0px 40px; text-align: justify;">“<i>From some informal interviews [we learned] people spend money on gruntwork—the stuff that's fun they're more likely to do anyway, money or not. </i>” <a href="https://twitter.com/b_vasilescu/status/1279199236094132227"><b>↗</b></a></p>
+<blockquote class="twitter-tweet tw-align-center" data-conversation="none"><p lang="en" dir="ltr">I can tell you from some informal interviews we did outside that paper, that people spend the money on gruntwork — the stuff that’s fun they’re more likely to do anyway, money or not.</p>&mdash; Bogdan Vasilescu (@b_vasilescu) <a href="https://twitter.com/b_vasilescu/status/1279199236094132227?ref_src=twsrc%5Etfw">July 3, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
Apologies if you used the original formatting for a reason, but I like how the blockquotes break up the post visually and show the profile pic of the author (studies have shown people like to see faces!)

Reply:
nope this is goood

Reply:
ah yeah this embed is perfect that's what I had in mind originally, thanks for the touch